ir/expression.cpp

```diff
@@ -125,6 +125,20 @@ IR::Constant::operator&(const IR::Constant &c) const {
   return IR::Constant(v);
 }
 
+IR::Constant
```

Comment:
all these operations ignore the types, but I guess they are patterned after the previous ones.